wpilibc/sim/src/RobotDrive.cpp

```diff
@@ -20,7 +20,8 @@ using namespace frc;
 
 const int RobotDrive::kMaxNumberOfMotors;
 
-static auto make_shared_nodelete(SpeedController* ptr) {
+static std::shared_ptr<SpeedController> make_shared_nodelete(
```

Comment:
I think you misunderstood me. Under C++14, the return type should have been `decltype(auto)` instead of `auto` since an rvalue is being returned. The corresponding return type would be `std::shared_ptr<SpeedController>&&`. To make this compile under C++11, replace `decltype(auto)` with that.